### DISCUSSION TITLE
add clusterrole to service account

### DISCUSSION THREAD

**Discussion by chabater** [okd-project/okd#794] [Q&A]
when adding clusterrole to a service, using 'oc adm policy add-cluster-role-to-user myclusterrole myserviceaccount,It doesnt make sense that it still work when there is typo in service account name

**Comment by duritong** [comment]
Adding a clusterrole just creates a role binding. The user in the rolebinding does not need to exist. It could be that the user exists on the future.

Therefor you can't detect a typo. And since creating the rolebinding was successful, the command execution is successful.
